outsource/meta.py

```diff
@@ -0,0 +1,47 @@
+# These data_types need per-chunk storage, so don't upload to rucio here!
+PER_CHUNK_DATA_TYPES = [
+    "peaklets",
+    "hitlets_nv",
+    "afterpulses",
+    "led_calibration",
+]
+
+
+# Do a query to see if these data_types are present
+DETECTOR_DATA_TYPES = {
+    "tpc": {
+        "raw": "raw_records",
+        "to_process": [
+            "peaklets",
+            "event_ms_naive",
```

Comment:
Why are event_info, peak_basics and event_area_per_channel removed?

Reply:
In the new logic, we will save `peak_basics` and `event_info` when processing `event_info_double`. And we do not allow `peak_basics` and `event_info` to be in `include_data_types` in `XENON_CONFIG`.

I will check with Lanqing about whether we saved `event_area_per_channel `.